src/renderer/shared/api/cross-chain/crossChainService.ts

```diff
@@ -22,3 +40,252 @@ export const getXcmConfig = (): XcmConfig | null => {
 export const saveXcmConfig = (config: XcmConfig) => {
   localStorage.setItem(XCM_KEY, JSON.stringify(config));
 };
+
+export const getAvailableDirections = (chains: ChainXCM[], assetId: number, chainId: ChainId): XcmTransfer[] => {
+  const chain = chains.find((c) => c.chainId === chainId);
```

Comment:
I think it's better to implement this feature on higher level, where we will have connection status, and here work only with xcm config

Reply:
In that case I think that a higher-level function for getting available Direction has to be implemented. Otherwise this method will be used directly that may lead to errors.